bunker_in_the_snow/map.conf

```
@@ -0,0 +1,18 @@
+team.1.pos = (103,-32,104)
+team.1.color = red
+team.2.pos = (-103,-32,-103)
+team.1 = red
+team.2 = blue
+team.2.color = blue
+
+initial_stuff = default:pick_stone,default:torch 10,shooter_guns:pistol_loaded

```

`pistol` and `pistol_loaded` both work fine, but `pistol` is better IMO.
```suggestion
initial_stuff = default:pick_stone,default:torch 10,shooter_guns:pistol
```

---

**No!** `shooter_guns:pistol` is unloaded pistol. I have tested it in many map tests and ik the output given to me.

---

https://github.com/MT-CTF/maps/pull/194#discussion_r563490287

---

Even this works fine...